torch/csrc/jit/backends/backend_init.cpp

```diff
@@ -179,6 +180,12 @@ void initJitBackendBindings(PyObject* module) {
         throw py::cast_error(c10::str(
             "Object ", py::str(orig_module), " is not a ScriptModule"));
       });
+
+  m.def(
+      "_jit_backend_generate_debug_handles",
+      [](std::shared_ptr<Graph>& graph) {
+         return generate_debug_handles(graph);
+      });
```

Comment:
This is going to change per the follow up discussed in PR/55462

So that PR should be done first.

Reply:
@raziel so my plan was to land all the diffs in this stack first and then do the PR that changes preprocess signature. That is a public API with few places where it might be used. And since it touches many sites, it may take longer to land. I did not want that to block progress on what we want to deliver this half.